packages/page-staking/src/Nomination/index.tsx

```diff
@@ -0,0 +1,380 @@
+// Copyright 2020
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import React, { useState, useEffect, useCallback } from 'react';
+import styled from 'styled-components';
+import BN from 'bn.js';
+import CreateModal from '@polkadot/app-accounts/Accounts/modals/Create';
```

Comment:
As explained earlier, no additional creates , the plan is to drop the key management in the webapp itself. (First deprecate, then drop - not overnight.) So no new additional dependencies, especially not on the Accounts app.